sparrow_datums/keypoints/keypoint_types.py

```diff
@@ -0,0 +1,51 @@
+"""Data types."""
+import enum
+
+import numpy as np
+import numpy.typing as npt
+
+FloatArray = npt.NDArray[np.float64]
+
+
+class PType(enum.Enum):
```

Comment:
Let's put all types in the original `PType` [enum](https://github.com/sparrowml/sparrow-datums/blob/1a3e630622782966bbdcf1a2a1a89c5d9c3c3dc9/sparrow_datums/types.py#L10) and delete this one.